packages/x-components/src/x-modules/ai/components/ai-overview.vue

```diff
@@ -29,45 +29,78 @@
             class="x-ai-overview-loading-content"
             data-test="ai-overview-loading-content"
           >
-            <span v-for="i in 4" :key="i" />
+            <span v-for="i in 4" :key="i" data-test="ai-overview-loading-item" />
           </div>
           <div v-else class="x-ai-overview-content" data-test="ai-overview-content">
-            <span>{{ question?.suggestionText }}</span>
-            <p>{{ question?.content.responseText }}</p>
+            <span>{{ suggestionText }}</span>
+            <p>{{ responseText }}</p>
           </div>
         </ChangeHeight>
       </div>
       <CollapseHeight
-        v-if="question?.content.searchQueries.length"
+        v-if="suggestionsSearch.length"
         :style="{
-          '--x-collapse-height-transition-duration': `${300 * question.content.searchQueries.length}ms`,
+          '--x-collapse-height-transition-duration': `${300 * suggestionsSearch.length}ms`,
         }"
+        data-test="ai-overview-collapse-height-suggestions"
       >
-        <div v-show="expanded" data-test="ai-overview-slot">
-          <slot :question="question">
-            <AiQuestionResults v-if="question" :question="question" />
+        <div v-show="expanded" data-test="ai-overview-suggestions-container">
+          <slot :suggestions-search="suggestionsSearch">
+            <div class="x-ai-overview-suggestions">
+              <div
+                v-for="{ query, results } in suggestionsSearch"
+                :key="query"
+                class="x-ai-overview-suggestion"
+              >
+                <QueryPreviewButton
```

Comment:
I insist whatever other solution could fix better than `QueryPreviewButton`. Using directly `BaseEventButton` o event a `button` with an emit, but this component is related with query-preview which is a feature itself